Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model loop_sm-no_b_mass
+
+define p = p b b~
+generate p p > t W- [QCD]
+add process p p > t~ W+ [QCD]
```

Comment:
Let's add @0 and @1 at the end of the two lines (i think this later helps you identify the cross sections separately after pythia run (if i remember correctly)

also change W+ W- to w+ w- (madgraph replaces upper cases to lower cases)